docs/networks/testnet/running-a-node.md

```diff
@@ -77,14 +80,122 @@ After the initialization is successful, we have to download the blockchain clien
 $ sudo lukso install
 ```
 
+#### Setting your Public IP Address
```

Comment:
let's not copy paste huge block of text around, maintenance will be harder for us.

I suggest to put:

Check mainnet docs and replace:

```
/testnet/prysm/pryms.yaml with /mainnet/prysm...
```

wdyt?
```